components/console_simple_init/README.md

```diff
@@ -32,6 +32,26 @@ It also provides an api to register an user provided command.
     // This allows you to execute the do_user_cmd function when the "user" command is invoked
     ESP_ERROR_CHECK(console_cmd_user_register("user", do_user_cmd));
 
+    // Register any other plugin command added to your project
+    ESP_ERROR_CHECK(console_cmd_all_register());
 
     ESP_ERROR_CHECK(console_cmd_start());    // Start console
     ```
+
+### Automatic registration of console commands
+`console_simple_init` component provides a function `console_cmd_all_register()` which performs automatic registration of all commands linked into the application.
+
+In order for the commands to be registered automatically, it has to place the registration function into `console_cmd_desc` section.
+
+To do that follow these steps:
```

Comment:
It should be clear that this part is only for implementing a new component with plug-able commands. This chapter may still give an impression that this is something users need to do to work with commands